<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3302F_cmd_utility v00.01</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QFrame" name="northFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QFrame" name="comPortFrame">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <item row="6" column="0">
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>COM:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="comComboBox">
            <item>
             <property name="text">
              <string>CLOSE</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="baudrateComboBox">
            <property name="editable">
             <bool>false</bool>
            </property>
            <property name="currentText">
             <string>1200</string>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>1200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4800</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>19200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>38400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>57600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>115200</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Baud Rate:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Flow control:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboBox_flow_ctl">
            <item>
             <property name="text">
              <string>No flow control</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Hardware flow control (RTS/CTS)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Software flow control (XON/XOFF)</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>StopBits</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Parity</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_6">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>DataBits</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="comboBox_stop_bits">
            <item>
             <property name="text">
              <string>OneStop</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>OneAndHalfStop</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>TwoStop</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="comboBox_parity">
            <item>
             <property name="text">
              <string>NoParity</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>EvenParity</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>OddParity</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SpaceParity</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MarkParity</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="comboBox_data_bits">
            <item>
             <property name="text">
              <string>Data5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Data6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Data7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Data8</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="toolButton">
         <property name="toolTip">
          <string>清除RS232訊息</string>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
         <property name="icon">
          <iconset resource="pictures/QComShell_Pic.qrc">
           <normaloff>:/new/prefix1/Gnome-Edit-Clear-48.png</normaloff>:/new/prefix1/Gnome-Edit-Clear-48.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="toolButton_refresh">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="pictures/QComShell_Pic.qrc">
           <normaloff>:/new/prefix1/1433159295_sync-01.png</normaloff>:/new/prefix1/1433159295_sync-01.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QCheckBox" name="checkBox_EnableTimeLog">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Display Time</string>
         </property>
         <property name="icon">
          <iconset resource="pictures/QComShell_Pic.qrc">
           <normaloff>:/new/prefix1/Time_48.png</normaloff>:/new/prefix1/Time_48.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="centerFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QSplitter" name="centerSplitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="handleWidth">
          <number>6</number>
         </property>
         <widget class="QFrame" name="leftFrame">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QSplitter" name="leftSplitter">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="handleWidth">
              <number>3</number>
             </property>
             <widget class="QPlainTextEdit" name="plainTextEdit_ascii">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="plainText">
               <string>請先下&quot;REMOTE&quot;命令使3302F進入REMOTE狀態
</string>
              </property>
             </widget>
             <widget class="QScrollArea" name="asciiScrollArea">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="asciiScrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>568</width>
                 <height>69</height>
                </rect>
               </property>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>檔案</string>
    </property>
    <addaction name="actionOpen_Command_List"/>
    <addaction name="actionSave_Command_List_2"/>
    <addaction name="separator"/>
    <addaction name="action_log"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>清除</string>
    </property>
    <addaction name="actionClean"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionBatchExec"/>
    <addaction name="actionAuto_save_log_file"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actionClean">
   <property name="text">
    <string>清除訊息</string>
   </property>
  </action>
  <action name="actionOpen_Command_List">
   <property name="text">
    <string>開啟指令清單</string>
   </property>
  </action>
  <action name="actionBatchExec">
   <property name="text">
    <string>批次執行</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_log">
   <property name="text">
    <string>儲存ASCII_Log檔案</string>
   </property>
  </action>
  <action name="actionSave_Command_List_2">
   <property name="text">
    <string>儲存指令清單</string>
   </property>
  </action>
  <action name="actionBin_Log">
   <property name="text">
    <string>儲存Bin_Log檔案</string>
   </property>
  </action>
  <action name="actionAuto_save_log_file">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto save log file </string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pictures/QComShell_Pic.qrc"/>
 </resources>
 <connections/>
</ui>
